spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -75,6 +75,7 @@ describe("Configuration loader", function()
     assert.same(false, conf.allow_debug_header)
     assert.same(KONG_VERSION, conf.lmdb_validation_tag)
     assert.is_nil(getmetatable(conf))
+    assert.are_equal(60000, conf.concurrency_timeout)
```

Comment:
should we use `assert.equal`?

Reply:
@chronolaw seems `equal` and `are_equal` both work.

Reply:
The above lines are using `assert.equal`, perhaps we should be consistent.

Reply:
Done.